core/src/main/java/org/elasticsearch/search/aggregations/AggregationBuilder.java

```diff
@@ -60,8 +65,33 @@ public B subAggregation(AbstractAggregationBuilder aggregation) {
     }
 
     /**
+     * Add a sub get to this bucket get.
```

Comment:
This was copied from the method above but makes no sense nonetheless. I wonder how long it's been there? I'll correct the JavaDocs for all of these methods and if necessary in SearchRequestBuilder and SearchSourceBuilder too